pkg/kubelet/cm/memorymanager/memory_manager_test.go

```diff
@@ -485,6 +484,8 @@ func TestGetSystemReservedMemory(t *testing.T) {
 }
 
 func TestRemoveStaleState(t *testing.T) {
+	logger, _ := ktesting.NewTestContext(t)
+	ctx := context.Background()
```

Comment:
```suggestion
	logger, tCtx := ktesting.NewTestContext(t)
```

The same below.

Reply:
Ah, I misinterpreted your previous comment. Fixed now, Thanks!